run_service.sh

```diff
@@ -382,14 +380,23 @@ try_read_storage() {
             fi
         done
 
+        unset USE_STAKING
+        unset AGENT_ID
```

Comment:
The 2 unsets here are to avoid using variables from the bash environment, instead from the source .env file in the next line.